src/WebApi/Application/UseCases/Question/DeleteInterviewQuestion/DeleteQuestionInput.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Application.UseCases.Question.DeleteInterviewQuestion
+{
```

Comment:
As Tomas mentioned, this should be a rather file-scoped namespace. You can change that through the contextual menu after right-clicking on namespace there is an auto-conversion tool.

If interested, here are the specs
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces